spec/Appendix A -- Notation Conventions.md

```diff
@@ -43,6 +43,8 @@ ListOfLetterA :
   - `a`
   - ListOfLetterA `a`
 
+The GraphQL grammar is greedy; this means that when given a choice between two definitions 
+in a production, the rule matching the longest sequence of tokens prevails. 
```

Comment:
> the rule matching the longest sequence of tokens prevails

"longest sequence" is ambigious, since it can mean number of tokens or number of symbols.
I think it better to state that optional tokens are always matched if possible and only then subsuequent rules are matched.

Reply:
I was wondering if I should add an example here. What do you think?

Reply:
Examples are always useful:
> The GraphQL spec, despite describing technical behavior, is intended to be read by people. Use natural tone and include motivation and examples.